draft-ietf-teas-actn-poi-applicability.md

```diff
@@ -263,25 +263,33 @@ IP link:
 
 > A link between two IP interfaces.
 
+PNC:
+
+> Provisioning Network Controller that provisions either the optical devices (O-PNC) or the packet devices (P-PNC).
```

Comment:
No need to redefine PNC since it is already defined in RFC8453 and the section stats referencing RFC8453 for the ACTN terminology